itou/templates/layout/_header_nav_primary_items.html

```diff
@@ -44,6 +44,7 @@
     </li>
     {% if user.is_authenticated %}
         {% if request.organizations|length > 1 %}
+            {% comment %} If the name of the organisation.display_name is longer than the .dropdown-structure width, it will be automatically truncated in css and it will be displayed in full in a tooltip. {% endcomment %}
```

Comment:
Pourquoi ne pas l’avoir mis proche du tooltip ? Ça augmente les chances d’expliquer pourquoi on fait le `{% if not is_mobile and org.display_name|length > 24 %}`.

Pour info, pour les commentaires sur une seule ligne, il est possible d’écrire : `{# Mon commentaire... #}`.

Reply:
Je l'ai mis juste au-dessus du `.dropdown-structure` parque le nom de la structure est tronqué en css a deux endroits a l'intérieur et "tooltipé" a un seul. C'était pour éviter de mettre deux commentaires à peu près similaires à deux endroits différents.
Pour le `{# comment... #}`, merci je corrige, je prefère aussi la version courte 👍